portfolio-html/fun.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Fun Things</title>
+  <LINK REL=StyleSheet HREF="/Users/brittanyjones/desktop/Adaprojects/personal-portfolio-site/stylesheets/normalize.css" TYPE="text/css" MEDIA=screen>
+  <LINK REL=StyleSheet HREF="/Users/brittanyjones/desktop/Adaprojects/personal-portfolio-site/stylesheets/fun.css" TYPE="text/css" MEDIA=screen>
+  <LINK REL=StyleSheet HREF="/Users/brittanyjones/desktop/Adaprojects/personal-portfolio-site/stylesheets/nav-footer.css" TYPE="text/css" MEDIA=screen>
+</head>
+
+<body>
+  <div>
+    <nav class="nav">
+      <ul class="navigation-list">
+        <li><a href="/Users/brittanyjones/desktop/Adaprojects/personal-portfolio-site/portfolio-html/portfoliohome.html">Home</a></li>
```

Comment:
You're using **absolute** links which **only** work on your computer.  You need to use links **relative** to the file you're linking from.  In this case you should do:

```html
  <li><a href="portfoliohome.html">Home</a></li>
  <li><a href="resume.html">Resume</a></li>
```